Applications/Custom/mae_loss.cpp

```diff
@@ -12,8 +12,57 @@
  */
 #include "mae_loss.h"
 
+#include <cmath>
+
+#include <tensor.h>
+
+constexpr const float EPSILON_ = 1e-7;
 namespace custom {
-const std::string MaeLossLayer::type = "mae_loss";
+
+int MaeLossLayer::initialize(nntrainer::Manager &manager) {
+  output_dim = input_dim;
```

Comment:
The parameter of this function, manager might be temporary.?

Reply:
Yes, manager will go away and context will take place.

Reply:
got it.